### ISSUE TITLE
Add non-alphabetic font-family similar to BLOKK or Redacted

### ISSUE THREAD

**Issue by erperion** [excalidraw/excalidraw#3635]
Just an idea. Currently there are tree fonts to choose and all of them are alphabetic so adding non-alphabetic might be useful for some folks. "Lorem ipsum" would look something like this "▮▮▮▮▮&nbsp;&nbsp;▮▮▮▮▮", but without space between letters.

Reference fonts:
[BLOKK](https://github.com/los-gordos/BLOKK)
[Redacted](https://github.com/christiannaths/redacted-font)

**Comment by ad1992** [comment]
Yeah, this can be helpful when designing mockups/wireframes. Let's see what the team thinks about it.

**Comment by dwelle** [comment]
adding more search keywords: placeholder text lorem ipsum squiggly lines

**Comment by Mrazator** [comment]
Awesome idea, we shall def. revisit in the future!
